google-cloud-bigquery/src/test/java/com/google/cloud/bigquery/it/ITBigQueryTest.java

```diff
@@ -1451,6 +1451,9 @@ public void testCreateAndGetTable() {
     assertNotNull(remoteTable.getLastModifiedTime());
     assertNotNull(remoteTable.<StandardTableDefinition>getDefinition().getNumBytes());
     assertNotNull(remoteTable.<StandardTableDefinition>getDefinition().getNumLongTermBytes());
+    assertNotNull(remoteTable.<StandardTableDefinition>getDefinition().getNumTotalLogicalBytes());
+    assertNotNull(remoteTable.<StandardTableDefinition>getDefinition().getNumActiveLogicalBytes());
+    assertNotNull(remoteTable.<StandardTableDefinition>getDefinition().getNumLongTermLogicalBytes());
```

Comment:
Note: We cannot add the rest of the newly added fields like TimeTravelBytes etc since they are not available for a table created few seconds ago.